launchable/commands/record/tests.py

```diff
@@ -442,3 +449,19 @@ def get_session_and_record_start_at_from_subsetting_id(subsetting_id: str, clien
         "session": "builds/{}/test_sessions/{}".format(build_number, test_session_id),
         "start_at": parse_launchable_timeformat(created_at)
     }
+
+
+def get_env_values(client: LaunchableClient) -> Dict[str, str]:
+    sub_path = "slack/notification/key/list"
+    res = client.request("get", sub_path=sub_path)
+
+    metadata = {}  # type: Dict[str, str]
+    if res.status_code != 200:
+        return metadata
+
+    keys = res.json()["keys"]
```

Comment:
[must]
This would raise error if there is no `keys` key in response json.
Better write like `keys = res.json().get("keys", [])` here.